lib/index.js

```diff
@@ -172,14 +172,16 @@ module.exports = ({
     ios: {
       enabled: unl(mergePlatforms.ios.enabled, true),
       assets: mergePlatforms.ios.assets,
+      projectPath: mergePlatforms.ios.projectPath,
     },
     android: {
       enabled: unl(mergePlatforms.android.enabled, true),
       assets: mergePlatforms.android.assets,
+      projectPath: mergePlatforms.android.projectPath,
     },
   };
 
-  const config = getConfig({ rootPath });
+  const config = getConfig({ rootPath, androidProjectPath: platforms.android.projectPath, iosProjectPath: platforms.ios.projectPath, });
```

Comment:
Breaking change for the node api (not cli) since now must provide paths for the projects